pysass.cpp

```diff
@@ -404,6 +435,116 @@ static void _add_custom_functions(
     sass_option_set_c_functions(options, fn_list);
 }
 
+Sass_Import_List _call_py_importer_f(
+        const char* path,
+        Sass_Importer_Entry cb,
+        struct Sass_Compiler* comp
+) {
+    PyObject* pyfunc = (PyObject*)sass_importer_get_cookie(cb);
+    PyObject* py_path = PyUnicode_FromString(path);
+    PyObject* py_args = PyTuple_New(1);
+    PyObject* py_result = NULL;
+    Sass_Import_List sass_imports = NULL;
+    Py_ssize_t i;
+    
+    PyTuple_SetItem(py_args, 0, py_path);
+    
+    if (!(py_result = PyObject_CallObject(pyfunc, py_args))) {
+        sass_imports = sass_make_import_list(1);
+        sass_imports[0] = sass_make_import_entry(path, 0, 0);
+        
+        PyObject* exc = _exception_to_bytes();
+        char* err = PySass_Bytes_AS_STRING(exc);
+        
+        sass_import_set_error(sass_imports[0],
+                              err,
+                              0, 0);
```

Comment:
should test this case probably.  The exception handling code was probably the hardest bit of the custom functions